src/Linker/Linker.cpp

```diff
@@ -45,15 +46,11 @@ enum class ProjectType
     NONE,
     FASTFILE,
     IPAK,
-
+    IWD,
     MAX
 };
 
-constexpr const char* PROJECT_TYPE_NAMES[static_cast<unsigned>(ProjectType::MAX)]{
-    "none",
-    "fastfile",
-    "ipak",
-};
+constexpr const char* PROJECT_TYPE_NAMES[static_cast<std::underlying_type_t<ProjectType>>(ProjectType::MAX)]{"none", "fastfile", "ipak", "iwd"};
```

Comment:
Unfortunately clang-format does not behave particularly cool when it comes to array initializer lists.

You can force it to make way less awkward formatting by adding a trailing comma `,` after the last entry which is what i do everywhere where i catch it until they introduce a better way to handle this :sweat_smile: